frontend/src/container/GroupedNotificationItem.jsx

```diff
@@ -0,0 +1,209 @@
+      else {
+        const content = notification.group.find(notification => notification.content).content
+        escapedContentLabel = escapeHtml(content.type === TLM_SUB.COMMENT
+          ? content.parentLabel
+          : content.label
+        )
+      }
+    }
+
+    const userList = uniqBy(notification.group.map(notification => notification.user), 'userId')
+    const escapedUser = userList.length > 1
+      ? { publicName: t('{{numberOfUsers}} user', { numberOfUsers: userList.length }) }
+      : escapeHtml(notification.group[0].user.publicName)
+
+    const i18nOpts = {
+      user: `<span title='${escapedUser}'>${escapedUser}</span>`,
+      author: `<span title='${escapedAuthor}'>${escapedAuthor}</span>`,
+      content: `<span title='${escapedContentLabel}' class='contentTitle__highlight'>${escapedContentLabel}</span>`,
+      numberOfContribution: notification.group.length,
+      numberOfWorkspaces: numberOfWorkspaces,
+      interpolation: { escapeValue: false }
+    }
+
+    if (numberOfContributionTypes > 1) {
+      let contentUrl
+      if (numberOfContents === 1) {
+        const content = notification.group.find(notification => notification.content).content
+        contentUrl = PAGE.CONTENT(content.type === TLM_SUB.COMMENT
+          ? content.parentId
+          : content.id
+        )
+      }
+
+      return {
+        text: numberOfWorkspaces > 1
+          ? t('{{author}} made {{numberOfContribution}} contributions in {{numberOfWorkspaces}} spaces', i18nOpts)
+          : numberOfContents === 1
+            ? t('{{author}} made {{numberOfContribution}} contributions on {{content}}', i18nOpts)
+            : t('{{author}} made {{numberOfContribution}} contributions', i18nOpts),
+        url: contentUrl
+      }
+    } else {
+      const notificationDetails = getNotificationDetails({
+        ...notification.group[0],
+        author: { publicName: escapedAuthor },
+        content: { ...notification.group[0].content, label: escapedContentLabel, parentLabel: escapedContentLabel },
+        numberOfWorkspaces: numberOfWorkspaces,
+        numberOfContents: numberOfContents,
+        type: notification.group[0].type,
+        user: { ...notification.group[0].user, publicName: escapedUser }
+      })
+
+      if (numberOfContents !== 1) notificationDetails.url = undefined
+      return notificationDetails
+    }
+  }
+
+  handleClickGroupedNotification = (e, notification) => {
+    const { props } = this
+
+    if (notification.type.includes(GROUP_NOTIFICATION_CRITERIA.CONTENT)) {
+      notification.group.forEach(notification => props.onClickNotification(e, notification.id, {
+        url: PAGE.CONTENT(notification.content.type === TLM_SUB.COMMENT
+          ? notification.content.parentId
+          : notification.content.id)
+      }))
+    } else {
+      props.dispatch(updateNotification(notification.id, notification.group))
```

Comment:
Are group notifications not updated, or updated somewhere else?
Can you explain why?

Reply:
if the notification is grouped by content, it's not updated on click (it just redirects to the content), I will add a INFO comment

Reply:
Ah, right. Feel free to had, or not, an info comment for this.